projects/1.20/assets/argentinasdelight/argentinas_delight/lang/zh_cn.json

```
@@ -0,0 +1,566 @@
+{
+  "block.argentinas_delight.aceitehirviendo": "一锅热油",
```

```suggestion
  "block.argentinas_delight.aceitehirviendo": "热油",
```
一般来说不需要加量词
如果必须要加容器，则参照原版对于桶之类的处理
```suggestion
  "block.argentinas_delight.aceitehirviendo": "热油锅",
```